### PR TITLE
Tests fixes

### PR DESCRIPTION
**Pull request by azat**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Changelog:
- tests: make 02152_http_external_tables_memory_tracking more stable
- tests: wait for left queries in 00417_kill_query
- clickhouse-test: increase timeout for obtaining processlist

### CODE REVIEW

**Comment by kitaisreal** [comment]
@azat need to fix style check. We added python tool for formatting.

File: tests/clickhouse-test [hunk 1]
```diff
@@ -116,7 +116,7 @@ def clickhouse_execute_http(base_args, query, timeout=30, settings=None, default
 def clickhouse_execute(base_args, query, timeout=30, settings=None):
     return clickhouse_execute_http(base_args, query, timeout, settings).strip()
 
-def clickhouse_execute_json(base_args, query, timeout=30, settings=None):
+def clickhouse_execute_json(base_args, query, timeout=60, settings=None):
```
Comment (kitaisreal) [inline tests/clickhouse-test:119]: @azat  why this can take more than 30 seconds ?

**Comment by azat** [comment]
>@azat need to fix style check. We added python tool for formatting.

This has been fixed multiple times already:
- #35754
- https://github.com/ClickHouse/ClickHouse/pull/35746/commits/f04756079eb5f125cc60296226a9ccfbe47f16e4

File: tests/clickhouse-test (hunk 1 above)
Comment (azat) [reply tests/clickhouse-test:119]: I guess due to parallel tests, for example [here](https://s3.amazonaws.com/clickhouse-test-reports/32928/ddd5bebe555ce8feebcdd339e47fc45184c20dd1/stateless_tests__thread__actions__[1/3].html), you can also find details in the commit message - https://github.com/ClickHouse/ClickHouse/pull/35745/commits/7d140c9a59c1760d7f89d6b1cbc3ef149f3324e5

**Review by kitaisreal — APPROVED** [review]

**Pull request merged**